launch/launch/main.py

```diff
@@ -22,25 +22,86 @@
 
 
 def file_exists(filename):
-    if not os.path.exists(filename) or not os.path.isfile(filename):
-        raise argparse.ArgumentError("'%s' does not exist" % filename)
-    return filename
+    return os.path.exists(filename) and os.path.isfile(filename)
+
+
+def get_launch_arg(arg, separator=":="):
+    # Make sure the separator exists
+    if separator not in arg:
+        return None
```

Comment:
These changes actually parse the `key:=value` pairs and creates the `arguments` dictionary that contains all of the mappings from key to value. Then the `launch()` functions just need to deal with a regular dictionary for argument checking.

I figured that the launch tool might be drastically different in the future. No worries if these changes don't get pulled in, or entirely removed at a later date.

Good catch on the redundant check, I will remove it.